src/ol/source/Vector.js

```diff
@@ -736,12 +736,23 @@ class VectorSource extends Source {
    * features.
    *
    * @param {import("../extent.js").Extent} extent Extent.
+   * @param {import("../proj/Projection.js").default} [projection] Projection.
    * @return {Array<import("../Feature.js").default<Geometry>>} Features.
    * @api
    */
-  getFeaturesInExtent(extent) {
+  getFeaturesInExtent(extent, projection) {
```

Comment:
The new `projection` argument should be optional. The documentation should state the purpose of this argument (i.e. needed for extents that cross the -180/180 line).

Reply:
Thanks. I've renamed it `opt_projection` and added to the documentation